shared/checks/oval/audit_rules_augenrules.xml

```diff
@@ -3,7 +3,8 @@
     <metadata>
       <title>Record Any Attempts to Run semanage</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_rhel-osp</platform>
```

Comment:
I don't think multi_platform_rhel-osp even works. Use the full name because there is only one in SSG.

Reply:
It works¸ the definition is in rhel-osp7 content. The build went from fail to success.
But yeah, there is only one OSP product.

Reply:
Updated.